modules/virt-deleting-vms.adoc

```diff
@@ -1,21 +1,23 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virt-delete-vms.adoc
+// * virt/managing-vms/virt-delete-vms.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-deleting-vms_{context}"]
 
 = Deleting a virtual machine by using the CLI
 
-You can delete a virtual machine by using the `oc` command-line interface (CLI). The `oc` client enables you to perform actions on multiple virtual machines.
+You can delete a virtual machine (VM) by using the `oc` command line interface (CLI). The `oc` client enables you to perform actions on multiple VMs.
+
+If the VM is delete protected, a message appears informing you that you must first disable the VM's delete protection before you can delete the VM.
 
 .Prerequisites
 
-* Identify the name of the virtual machine that you want to delete.
+* Identify the name of the VM that you want to delete.
```

Comment:
I know this was already here, but this is kind of an odd prereq. I think you could delete it.